src/hub.js

```diff
@@ -225,6 +229,11 @@ async function handleHubChannelJoined(entryManager, hubChannel, data) {
 
   console.log(`Scene URL: ${sceneUrl}`);
   const environmentScene = document.querySelector("#environment-scene");
+  const objectsScene = document.querySelector("#objects-scene");
```

Comment:
Still a bit concerned about having all the pinned entities living in a few layers of wrapper object, surprised it hasn't caused more issues..

Reply:
it had some issues :) but i fixed the ones I found